src/pages/Sheet.tsx

```diff
@@ -1,17 +1,21 @@
-import React, { Component } from "react";
-import { Cittadino } from "../components/Cittadino";
+import React, {useEffect} from "react";
+import Citizen from "../components/Citizen";
 import { ListaTransazioni } from "../components/ListaTransazioni";
 
-export class Sheet extends Component {
-  public render() {
-    return (
-      <>
-        <Cittadino />
 
-        <hr />
 
-        <ListaTransazioni />
-      </>
-    );
-  }
+
+function Sheet(props) {
+
+  return (
+    <>
+      <Citizen {...props} />
+
+      <hr />
+
+      <ListaTransazioni />
```

Comment:
please use english for names

Reply:
I 'll solve this task and all the other one related to "transazioni" in another specific PR :)